test/parallel_api/ranges/rotate_view.pass.cpp

```diff
@@ -36,8 +36,9 @@ main()
 
     using namespace oneapi::dpl::experimental::ranges;
 
-    auto view1 = nano::ranges::views::all(data) | views::rotate(rotate_val);
-    auto view2 = views::rotate(nano::ranges::views::all(data), rotate_val);
+    //the name nano::ranges::views::all is not injected into oneapi::dpl::experimental::ranges namespace
```

Comment:
I would say not "not injected" but instead "not available" because, "not available" is the design decision that we don't make those names available in any of oneDPL namespaces but "no injected" just say the technical reason and reveals implementation details. I mean the PR looks good and I am not insisting on this change at all. I hope those thoughts would help up to make clearer comments in the future